.github/workflows/build_main_documentation.yml

```diff
@@ -72,8 +72,11 @@ jobs:
           pip install .
           cd ..
 
-          cd ${{ inputs.package }}
-          pip install .[dev]
+          if [ "${{ inputs.additional_args }}" != *"--not_python_module"* ]
```

Comment:
This checks that `--not_python_submodule` is not a substring of `inputs.additional_args`

Reply:
Ah, it turns out you need to use double brackets with the wildcard outside the string: see [here](https://linuxize.com/post/how-to-check-if-string-contains-substring-in-bash/#using-wildcardshttps://linuxize.com/post/how-to-check-if-string-contains-substring-in-bash/#using-wildcards) and [here](https://stackoverflow.com/questions/229551/how-to-check-if-a-string-contains-a-substring-in-bash). I've fixed this in https://github.com/huggingface/doc-builder/pull/154/commits/2eb9af4659112886f9e52fcad439e2461771d365

Reply:
Just to be sure, can you run a test on this branch of the doc-builder to build the course doc before merging?

Reply:
Sure, good idea!